.github/workflows/CreatePullRequest.yml

```diff
@@ -0,0 +1,21 @@
+name: Create Pull Request
+
+on:
+  pull_request:
+    types: ["opened"]
+
+jobs:
+  CreateCardForStandalonePR_job:
+    name: Assign PR to the author and create a Kanban card
+    runs-on: ubuntu-latest
+    # Single quotes must be used here https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#literals
+    # PRs from forks don't have required token authorization
+    # Dependabot works directly under our repository, but doesn't have enough priviledges to create project card
```

Comment:
Do we need the comments? I don't understand the link between dependabot and this pr.

Reply:
Unfortunately, we do. I've been bitten hard by these 😒 

Dependabot creates PRs. That triggers this `CreatePullRequest.yml` and causes troubles with the action, because there's special context and permissions. It explains what the hell is going on on line 16. In-line comment would do better job, but it would be too long